TINY-7884: added docs for the new rtc_unsupported_content_notification setting (#2067)
tinymce/tinymce-docs: rtc/configuration.md

The same as [`rtc_client_connected`](#rtc_client_connected)

{% include rtc/client-connect-disconnect-example.md %}

### `rtc_unsupported_content_notification`

> The `rtc_unsupported_content_notification` option can be used to disable the notification about unsupported content inside the editor. Existing documents that has content generated by plugins that are unsupported by rtc could potentially break.

Author: Unsure how strong we want this message to be.

> The `rtc_unsupported_content_notification` option can be used to disable the notification about unsupported content inside the editor. Existing documents that has content generated by plugins that are unsupported by rtc could potentially break.

Reviewer: ```suggestion
The `rtc_unsupported_content_notification` option can be used to disable the notification about unsupported content inside the editor. Existing documents contain content generated using plugins that are unsupported by RTC plugin could lead to unexpected behavior.
```

Author: Thanks much better.

> The `rtc_unsupported_content_notification` option can be used to disable the notification about unsupported content inside the editor. Existing documents contain content generated using plugins that are unsupported by RTC plugin could lead to unexpected behavior.

Reviewer 2: ```suggestion
The `rtc_unsupported_content_notification` option can be used to disable the notification about unsupported content inside the editor. Existing documents containing content generated using plugins that are unsupported by RTC plugin could lead to unexpected behavior.
```

> **Type:** `boolean`

Reviewer 2: ```suggestion
**Type:** `Boolean`
```
Everything else in the docs/this file use this format generally :shrug:

> **Default:** true

Reviewer: ```suggestion
**Default:** `true`
```

**Required:** no

> #### Example of providing dynamic content from the server

Reviewer 2: This heading doesn't look correct? I assume it was copied and not updated.
